redirects-list.json

```diff
@@ -92,6 +92,9 @@
   "^/doc/command-reference/experiments$                                                   /doc/command-reference/exp",
   "^/doc/command-reference/ls$                                                            /doc/command-reference/list 302",
 
+  "^/doc/dvclive/dvclive-with-dvc$                                                        /doc/dvclive/get-started",
+  "^/doc/dvclive/ml-frameworks$                                                           /doc/dvclive/api-reference/ml-frameworks",
```

Comment:
Update on https://github.com/iterative/dvc.org/pull/3923#issuecomment-1239774786 (relates to the redirects above):

<img width="551" alt="image" src="https://user-images.githubusercontent.com/1477535/197704906-ffbe0b77-85d8-488b-9e78-482b7419fe05.png">

> Looks like frameworks and DVC integration trump the Get Started quite a bit.

Traffic in general is up, esp. to the DVClive's home page and somewhat to the Get Started (👍🏼). But he ML frameworks and DVC integration pages have less traffic now 🤷🏼

Reply:
p.s. I just realized it's only been a week since this was merged so let's check again later.